packages/vuetify/src/composables/nested/__tests__/selectStrategies.spec.ts

```diff
@@ -731,5 +765,51 @@ describe('selectStrategies', () => {
         ['6', 'on'],
       ]))
     })
+
+    it('should not select disabled children when parent is selected', () => {
```

Comment:
return-object is usually vulnerable when nested composable has changes, so probably better add a test case to cover return-object as well